include/algorithms/gradient_boosted_trees/gbt_classification_training_types.h

```diff
@@ -69,6 +69,17 @@ enum LossFunctionType
     custom        /* custom function type */
 };
 
+enum OptionalResultNumericTableId
+{
+    weight = classifier::training::lastResultId + 1,
```

Comment:
were this namings agreed in LLD?
Not informative that it`s related with feature importance.

Reply:
Agree, variableImportanceWeight looks bette

Reply:
Fixed

Reply:
Changes should be added to documentation also.